prose/views.py

```diff
@@ -1,11 +1,40 @@
 from django.core.files.storage import default_storage
 from django.http import JsonResponse
 
+from uuid import uuid4
+from datetime import datetime
+
+from django.conf import settings
+
+import filetype
+
+
+def validate_file(file):
```

Comment:
This PR looks like a nice contribution!

Instead of multiple if/then statements that check for attributes and then set variables, can I suggest using `getattr` to simplify the logic into the following:

```python
def validate_file(file):
    allowed_file_types = getattr(
        settings,
        "PROSE_ATTACHMENT_ALLOWED_FILE_TYPES",
        [
            "image/jpeg",
            "image/png",
            "image/gif",
        ],
    )
    allowed_file_size = getattr(settings, "PROSE_ATTACHMENT_ALLOWED_FILE_SIZE", 5)
    file_type = filetype.guess(file).mime
    file_size = file.size / 1024 / 1024
    if file_type in allowed_file_types and file_size <= allowed_file_size:
        return True
    return False
```

Reply:
Great idea, thank you! I've made the change.